Pages/Authentication/logout.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Logout</title>
+    <link
+      rel="stylesheet"
+      href="
+    https://eagleui.netlify.app/css/component.css"
+    />
+    <link rel="stylesheet" href="./auth.css" />
+  </head>
+  <body>
+    <main class="logout__container">
+      <section class="logout">
+        <div class="logout__content">
+          <h3 class="hero__heading">You are logged out</h3>
+          <p class="hero__sub__heading">
+            You are logges out. To see your account please log in.
+          </p>
+          <p class="hero__subheading">
+            Don't have an account? Sign in to create one
+          </p>
+
+          <div class="logout__buttons">
+            <a class="btn btn__default">Login</a>
+            <a class="btn btn--outlined outlined__secondary">Signin</a>
```

Comment:
Just a suggestion, here you can give signup instead of signin as login and signin have the same meaning

Reply:
Ohh thanks for pointing out